HISTORY.rst

```diff
@@ -2,6 +2,29 @@
 History
 =======
 
+3.4.0 (2020-01-09)
+------------------
+
+Changes:
+
+* Added ``activate_flow`` task which can be used to activate Flows and Process Builder processes.
+
+* Added two tasks, ``disable_tdtm_trigger_handlers`` and ``restore_tdtm_trigger_handlers``, which can be used to disable trigger handlers for the table-driven trigger management feature of NPSP and EDA.
+
+* In the ``load_dataset`` task, added a way to avoid resetting the Salesforce Id mapping tables by setting the ``reset_oids`` option to False. This can be useful when running the task multiple times with the same org.
+
+* Added support for a few new metadata types from API versions 47 and 48 in the ``update_package_xml`` task.
+
+* Added a way for Robot Framework libraries to register custom locators for use by the selenium library.
```

Comment:
```suggestion
* Added a way for Robot Framework libraries to register custom locators for use by the Selenium library.
```

Should we provide some kind of a pointer to where this functionality is located/how to use it? All of the other line items for new functionality have a name to them.

Reply:
@davidmreed yes, but at the moment the only documentation about it is in @boakley's RFC, so there isn't something to point at